tests/src/rules/match-default-export-name.js

```diff
@@ -85,6 +85,34 @@ ruleTester.run('match-default-export-name', rule, {
         },
       ],
     }),
+    test({
+      code: 'import someComponentStyles from "./match-default-export-name/some-component.module.css";',
+      options: [
+        {
+          overrides: [
+            {
+              module: '/([\\w-]+)\\.module\\.css$/',
+              name: ['$1Styles'],
+              transform: 'camelCase',
+            },
+          ],
+        },
+      ],
+    }),
+    test({
+      code: 'import someComponentStyles from "./match-default-export-name/some-component.module.css";',
```

Comment:
```suggestion
      code: 'import style from "./match-default-export-name/some-component.module.css";',
```

чтоб и такой кейс проверить